libs/core/multi-input/multi-input.component.html

```diff
@@ -48,8 +48,9 @@
             [isControl]="true"
             [glyph]="showAddonButton ? glyph : ''"
             [glyphFont]="glyphFont"
-            [addonButtonAriaHidden]="!!title"
+            [addonButtonAriaHidden]="addonButtonAriaHidden"
```

Comment:
<img width="1470" alt="image" src="https://github.com/user-attachments/assets/ef32abf8-cf98-4a66-a788-db9dea02c369">

We have a common issue throughout the application where we use aria-hidden on focusable elements or elements with focusable children, which isn't allowed. In this case, the button is focusable and causes the error shown in the screenshot. I fixed a similar issue in my PR: https://github.com/SAP/fundamental-ngx/pull/12445.
<img width="806" alt="image" src="https://github.com/user-attachments/assets/6d8609f1-ce95-4277-965c-85e514e5d6cd">